tools/dependencies-report/src/main/resources/notices/gradle-license-report-0.7.1-NOTICE.txt

```diff
@@ -0,0 +1,201 @@
+                               Apache License
```

Comment:
I find it interesting that a project whose sole purpose is to report on licenses fails to correctly apply the license to itself by following the instructions in the Apache License's "How to apply the Apache License to your work".